air/Cargo.toml

```diff
@@ -24,6 +24,7 @@ crypto = { version = "0.6", path = "../crypto", package = "winter-crypto", defau
 fri = { version = "0.6", path = "../fri", package = "winter-fri", default-features = false }
 math = { version = "0.6", path = "../math", package = "winter-math", default-features = false }
 utils = { version = "0.6", path = "../utils/core", package = "winter-utils", default-features = false }
+libm = "0.2.8"
```

Comment:
minor nit: could we put dependencies in the alphabetical order?

Reply:
Sure!